src/identity/utils/convertIdentityToMe.ts

```diff
@@ -20,11 +20,14 @@ export const convertIdentityToMe = (
     accountType: account.type,
     paygCreditStartDate: account.paygCreditStartDate,
     billingProvider: account.billingProvider ? account.billingProvider : null,
+    // isRegionBeta in quartz/me has the opposite value of isUpgradeable.
+    isRegionBeta: account.isUpgradeable
```

Comment:
I'll defer to @randycoulman on the relationship, but here's my understanding:

I believe we can use `isUpgradeable` and `isRegionBeta` as the inverse of each other because, even though the property is populated for account account, the sole function performed by this property is to validate whether or not the user can go to `checkout` to upgrade a free account. 

I believe you're 100% correct that these properties aren't mirror images insofar as a PAYG or contract account could have `isRegionBeta === false`. After this PR, that PAYG or contract account will have `isRegionBeta === true`. But AFAIK this does not change the functionality of the app. `isRegionBeta` is only used as a check for showing the upgrade button (and controlling access to the checkout page). But the user also can't see the upgrade button unless the account type is `free`. If the sole purpose for which `isRegionBeta` is used is to prevent users in _free_ accounts from accessing checkout if they're in a beta region, the property isn't performing any function for PAYG or contract accounts anyway.

![Screen Shot 2022-09-19 at 11 20 53 AM](https://user-images.githubusercontent.com/91283923/191053815-ef2907cc-26fe-47fb-8971-b5ed47fcf886.png)

Another way to do this would be to 'hold off' on removing the extra network call, and simply roll this change into the removal of the `quartzMe` properties. I think this is a little more gradual though, as it lets us ensure this property is working correctly on `/identity` before we go ahead and remove the `quartzMe` code.

Reply:
I guess my point is: we could just change all the references in the code to use `isUpgradeable` now. But doing it this allows us to retain the ability to toggle the `identity`-related changes off if something goes awry.

Reply:
`isUpgradeable` is essentially the same as the old check around the appearance of the upgrade button. An account is upgradeable if it is a free account whose (single) org is in a GA (non-beta) region.

With multi-org, we have to change that to be a free account that doesn't have any orgs in beta regions.

If the only place that `isRegionBeta` is used is to control the visibility of the `Upgrade Now` button and access to the checkout page, then this approach seems like a reasonable thing to do for the (hopefully brief) transition period from old API to new.

If it was going to live longer than that, then I'd share the same concerns as @hoorayimhelping here, because they are not true opposites in that sense.